### PR TITLE
feat: New react frontend

### PR DESCRIPTION
**Pull request by sasjo**
This PR introduces a new frontend and build process for the tech radar.

### CODE REVIEW

File: src/js/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import radarService from '../modules/radarService';
+import Icon from './Icon';
+
+const Dot = () => (
+  <React.Fragment>
+    {' '}
```
Comment (tozz) [inline src/js/components/Footer.jsx]: Can't this be combined in a template string?

File: src/js/modules/radarService.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { firstBy } from 'thenby';
+import { pick } from './utils';
+
+// Do we need the fetch-polyfill?
```
Comment (tozz) [inline src/js/modules/radarService.js]: Nope

File: src/js/components/Entry.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  related: PropTypes.arrayOf(PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    file: PropTypes.string.isRequired,
+  })),
+};
+Related.defaultProps = {
+  related: [],
+};
+
+export const History = (props) => {
```
[the remark this answers is not in the document]
Comment (sasjo) [reply src/js/components/Entry.jsx]: Moved them to the `entry` sub dir and split the test too.

File: src/js/components/Entry.jsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      <ul>
+        {related.map(value => (
+          <li key={value.file}>
+            <Link to={value.file}>{value.name}</Link>
+          </li>
+        ))}
+      </ul>
+    </React.Fragment>
+  );
+};
```
[the remark this answers is not in the document]
Comment (sasjo) [reply src/js/components/Entry.jsx]: Added som spacing, but not between subsequent const declarations.

**Pull request merged**